Tools/autotest/quadplane.py

```diff
@@ -930,7 +930,6 @@ def tests(self):
 
         ret = super(AutoTestQuadPlane, self).tests()
         ret.extend([
-            self.AirMode,
```

Comment:
Can you provide a link to an instance of this failing, please?

Reply:
https://github.com/ArduPilot/ardupilot/tree/Copter-4.3

These are the two tests failing in Copter 4.3.